src/navigator/Screens.tsx

```diff
@@ -49,13 +49,15 @@ export enum Screens {
   MoonPayScreen = 'MoonPayScreen',
   XanpoolScreen = 'XanpoolScreen',
   NameAndPicture = 'NameAndPicture',
+  OnboardingRecoveryPhrase = 'OnboardingRecoveryPhrase',
   OnboardingSuccessScreen = 'OnboardingSuccessScreen',
   OutgoingPaymentRequestListScreen = 'OutgoingPaymentRequestListScreen',
   PaymentRequestConfirmation = 'PaymentRequestConfirmation',
   PaymentRequestConfirmationLegacy = 'PaymentRequestConfirmationLegacy',
   PhoneNumberLookupQuota = 'PhoneNumberLookupQuota',
   PhotosEducation = 'PhotosEducation',
   PhotosNUX = 'PhotosNUX',
+  ProtectWallet = 'ProtectWallet',
```

Comment:
nit: most of these are in alphabetical order-- we should probly try to keep it that way